integration-tests/bats/import-append-tables.bats

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+}
+
+teardown() {
+    assert_feature_version
+    teardown_common
+}
+
+@test "import-append-tables: disallow overwriting row during append" {
```

Comment:
These tests should verify the contents of the table after the import, not just the duplicate error handling

Reply:
Fixed.